src/handlers/ground-truths/chat-bot.ts

```diff
@@ -0,0 +1,64 @@
+import { Context } from "../../types";
+import { logger } from "../../helpers/errors";
+
+export async function fetchRepoDependencies(context: Context) {
+  const {
+    octokit,
+    payload: {
+      repository: {
+        owner: { login: owner },
+        name: repo,
+      },
+    },
+  } = context;
+
+  const { data: packageJson } = await octokit.repos.getContent({
+    owner,
+    repo,
+    path: "package.json",
+  });
+
+  if ("content" in packageJson) {
+    return extractDependencies(JSON.parse(Buffer.from(packageJson.content, "base64").toString()));
+  } else {
+    throw logger.error(`No package.json found in ${owner}/${repo}`);
+  }
+}
+
+export function extractDependencies(packageJson: Record<string, Record<string, string>>) {
```

Comment:
Broke this into a fn to make it easier to see what we are pulling from the `package.json`.

I'm thinking it might have value to also include at least some of these, lmk which ones and if you have a prompt structure in mind also lmk.


```json
{
"name": "@ubiquity-os/command-ask",
  "version": "1.0.0",
  "description": "A highly context aware organization integrated chatbot",
  "author": "Ubiquity DAO",
  "license": "MIT",
  "main": "src/worker.ts",
  "engines": {
    "node": ">=20.10.0"
  },
"keywords": [
    "typescript",
    "template",
    "dao",
    "ubiquity",
    "open-source"
  ],
```